super-productivity: 7.6.0 -> 7.9.1

LeUlukai opened the pull request:
<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!
List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

###### Motivation for this change

There is a new version of super productivity.
Electron 11 is marked as insecure; therefore, I have updated the used version to electron 12.

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

LeUlukai committed:
super-productivity: 7.6.0 -> 7.8.0

sikmir commented on pkgs/applications/office/super-productivity/default.nix:
```diff
@@ -1,15 +1,15 @@
 
 let
-  electron = electron_11;
+  electron = electron_12;
```
Would be better to move it to top level:
```
  super-productivity = callPackage ../applications/office/super-productivity {
    electron = electron_12;
  };

```

LeUlukai replied:
Done :-)

bobby285271 commented:
[Electron 12 has reached end-of-support](https://www.electronjs.org/blog#end-of-support-for-12xy), and there is [a PR that is trying to move packages away from electron 12](https://github.com/NixOS/nixpkgs/pull/148182), is it possible to use a new version of electron for this package (not tested myself)?

LeUlukai commented:
@bobby285271 Good point, I am going to test using a newer version of electron for super productivity.

LeUlukai committed:
super-productivity: use electron_13

LeUlukai committed:
super-productivity: 7.8.0 -> 7.9.1

LeUlukai commented:
Using Electron 13 seems to work seamlessly; thus, I have changed it to use `electron_13`.

In the meantime, Super Productivity received some updates; therfore, I have added a commit to update it to the current version (7.9.1).

bobby285271 commented:
@ofborg build super-productivity
